src/components/features/SaleBox/SaleBox.module.scss

```diff
@@ -0,0 +1,106 @@
+@import "../../../styles/settings.scss";
+
+.wrapper {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  min-height: 400px;
+  padding-right: 30px;
+
+  .leftinner {
+    width: 50%;
+    height: 400px;
+    color: $text-color-dirty-white;
+
+    img {
+      width: 100%;
+    }
+
+    .box_img::before {
+      position: absolute;
+      content: "";
+      opacity: 0.4;
+      background-color: $color-black;
+      width: 45%;
+      height: 65%;
+      top: 20px;
+      left: 20px;
+    }
+
+    .promo_text {
+      font-size: 28px;
+      bottom: 50%;
+      left: 1%;
+      padding-left: 25px;
+
+      span {
+        display: block;
+        font-size: 36px;
+        font-weight: 700;
+      }
+    }
+
+    .text_price {
+      color: $text-color-dirty-white;
+      bottom: 30%;
+      left: 3%;
+      font-size: 36px;
+      font-weight: 700;
+      border: 5px solid $text-color-dirty-white;
+    }
+  }
+  .rightinner {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    height: 405px;
+    padding-left: 30px;
+    img {
+      width: 540px;
```

Comment:
takie ustawienie szerokości na sztywno będzie sprawiało problemy. Lepiej dać `  width na 100% `. Do tego w ` .wrapper ` masz niepotrzebnie dodany ` padding-right `. Jak go tam usuniesz to zdjęcia się wrównają

Reply:
Poprawione